umbraco-heartcore/getting-started/preview.md

```diff
@@ -8,6 +8,8 @@ The Preview API is always protected, this means it requires an API Key with **Br
 
 You will also need a client consuming the Content Delivery API. If you don't already have one you can use one of the samples included with the Client Libraries.
 
+This preview feature is currently available only on the Professional Plan.
```

Comment:
How about adding a link to the Pricing page here, using "Professional Plan"?

Also, if this is added to a note it would be highlighted a little more. Would that make sense?